github/resource_github_team_members.go

```diff
@@ -64,9 +63,9 @@ func resourceGithubTeamMembersCreate(d *schema.ResourceData, meta interface{}) e
 	orgId := meta.(*Owner).id
 
 	teamIdString := d.Get("team_id").(string)
-	teamId, err := strconv.ParseInt(teamIdString, 10, 64)
+	teamId, err := getTeamID(teamIdString, meta)
```

Comment:
I have the same concern as @csainty [here](https://github.com/integrations/terraform-provider-github/pull/1502#issuecomment-1418282281) This is making additional call and I'm already hitting api limits. 
Maybe this can try to parse the teamId and if it fails look it up by slug, to avoid adding additional api calls

Reply:
@ilmax Agreed, that seems like the right thing to do here.  @dennislapchenko is that something you'd be up for changing in the 4 calls here?

Reply:
Sure, @nickfloyd ! I thought y'all agreed it is not a real issue since that PR got merged.
I have committed and update to `getTeamID` func. It does make sense to remove API call when an integer is passed, since github would rightfully return an error if the team does not exist.